docs/source/guide/scripts.md

```diff
@@ -0,0 +1,211 @@
+
+Custom scripts are a powerful tool to help you fully customize your labeling workflow. However, in doing so you are running arbitrary JavaScript code on each annotator’s machine, which comes with certain risks. Because arbitrary JS can be written and run with this tool, so we can give no guarantees of safety or appropriateness. 
+
+For this reason, you must opt-in before enabling custom scripts for your organization, and we urge you to use this feature with caution. 
+
+To enable custom scripts for your organization, you cannot have members that are in multiple organizations. This is enforced through application logic and is necessary for data security. The most common reason for this is when users have accounts in an expired free trial, but can also happen if you are using multiple organizations for project management or if you have an initial proof of concept or testing org. 
+
+## Label Studio Interface (LSI)
+
+The Label Studio Interface (LSI) is a helper object that is designed to be used with custom scripts. 
+
+LSI simplifies access to some data and can perform special actions that only make sense within the framework of custom scripts.
+
+### Instance methods
+
+##### `import(url, integrity)`
+
+Allows loading additional external scripts
+
+| Parameter  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Type     | Description          |
+|--------------------------|--------------------|--------------------------|
+| `url` | string | Specifies the URL of an external script file.                                 |
+| `integrity`  | string | Allows a browser to check the fetched script to ensure that the code is never loaded if the source has been manipulated.  |
+
+The method is asynchronous, so you can wait for the script to load before performing the main actions. For example:
+
+```javascript
+await LSI.import('<https://cdn.plot.ly/plotly-2.26.0.min.js>', 'sha384-xuh4dD2xC9BZ4qOrUrLt8psbgevXF2v+K+FrXxV4MlJHnWKgnaKoh74vd/6Ik8uF');
+console.log("Plotly is ready");
+```
+
+##### `on(eventName, handler)`
+
+Subscription to listen to events related to the Label Studio Frontend. All handlers added within the framework of this method will be unsubscribed when switching to another annotation.
+
+For a list of all available events, see our [Frontend reference](frontend_reference#Available-events).
+
+| Parameter &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Type     | Description          |
+|--------------------------|--------------------|--------------------------|
+| `eventName` | string | A case-sensitive string representing the event type to listen for.                  |
+| `handler`  | function | Allows a browser to check the fetched script to ensure that the code is never loaded if the source has been manipulated.  |
```

Comment:
This is wrong - was copied wrong in the source. What do we need to put here instead?